README.md

```diff
@@ -1,32 +1,190 @@
-# SQL Create Statement
+# SQL CREATE TABLE, DROP TABLE
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Create a new database
+- Create a new table
+- Drop a table
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+The SQL `CREATE TABLE` statement creates a new table
+by specifying the table name and the column details, including
+column name and data type.    The `DROP TABLE` statement deletes a table
+from the database.
 
-## Topic 1
+## Create A New Database (Code-along)
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+We will create a new database to store information about pets and their owners.
 
-## Topic 2
+1. Launch the **pgAdmin** tool.
+2. Enter the password for the **postgres** user.
+3. You should see a default server **PostgreSQL 15** along with a database named **postgres**:   
+  
+![pgadmin default server](https://curriculum-content.s3.amazonaws.com/6002/sql-create-statement/pgadmin.png)
+    
+4. Right-click on the `Databases` icon and select Create > Database:      
+  
+![new database](https://curriculum-content.s3.amazonaws.com/6002/sql-create-statement/newdb.png)
+    
+5. Name the new databases "pets" and press Save.    
+     
+![name pet database](https://curriculum-content.s3.amazonaws.com/6002/sql-create-statement/namepetsdb.png)
+    
+6. Confirm the new database named `pets` appears in the server view:    
+     
+![new pet database](https://curriculum-content.s3.amazonaws.com/6002/sql-create-statement/newpetdb.png)
+
+
+We will use the **Query Tool** to execute SQL statements.
+
+1. Click on the `pets` database to select the database.
+2. Click on the `query tool` icon.
+ 
+![open query tool](https://curriculum-content.s3.amazonaws.com/6002/sql-create-statement/openquerytool.png)
+
+The query tool opens a panel where we can write SQL statements.
+Confirm the connection is for  "pets/postgres@PostgreSQL 15". 
+
+![query tool](https://curriculum-content.s3.amazonaws.com/6002/sql-create-statement/querytool.png)
+
+
+
+## CREATE TABLE
+
+The `CREATE TABLE` statement creates a new table
+by specifying the table name and the column details.
+Each column is assigned a name and data type, along with other optional attributes.
+
+While PostgreSQL supports dozens of data types, we will primarily work with the following:
+
+- INTEGER stores integer values
+- DECIMAL stores fractional numeric values
+- TEXT stores textual values
+- DATE stores year, month, and day
```

Comment:
Should we specify the date format here too? (yyyy-MM-dd)?

Reply:
I might also be getting ahead of myself, but should we show them other data types too (LONG, FLOAT, VARCHAR, DATETIME, BLOB, etc.)? Or is that too much to throw at students all at once just learning SQL?

Or maybe we could add a link saying something like "To explore more data types, please see [PostgreSQL Data Types](https://www.postgresql.org/docs/current/datatype.html)."

Reply:
I tried to stick with a few basic types that were used in the existing SQL lessons to avoid
confusion with char, varchar, text, etc.  I added the link you recommended.